src/main/java/com/toptrumps/core/Dealer.java

```diff
@@ -1,4 +1,34 @@
 package com.toptrumps.core;
 
+import com.sun.xml.internal.bind.v2.TODO;
+
 public class Dealer {
+
+    private DeckParser deckParser;
+    private Game game;
```

Comment:
We should not instantiate the Game inside Dealer. According to the UML Class Diagram, it was supposed to be the other way around.